src/lib/spinel/spinel.c

```diff
@@ -1272,6 +1272,7 @@ const char *spinel_prop_key_to_cstr(spinel_prop_key_t prop_key)
         {SPINEL_PROP_MAC_CCA_FAILURE_RATE, "MAC_CCA_FAILURE_RATE"},
         {SPINEL_PROP_MAC_MAX_RETRY_NUMBER_DIRECT, "MAC_MAX_RETRY_NUMBER_DIRECT"},
         {SPINEL_PROP_MAC_MAX_RETRY_NUMBER_INDIRECT, "MAC_MAX_RETRY_NUMBER_INDIRECT"},
+        {SPINEL_PROP_MAC_CSL_ACCURACY, "MAC_CSL_ACCURACY"},
```

Comment:
Should this be SPINEL_PROP_RCP_CSL_ACCURACY?

Reply:
Changed.